airbyte-integrations/connectors/destination-snowflake/metadata.yaml

```diff
@@ -50,4 +36,113 @@ data:
   supportsDbt: true
   tags:
     - language:java
+  connectorTestSuitesOptions:
```

Comment:
❓ Wow. This is more individual secrets than I expected. Are we sure they are all actually used? Did we cross reference them with the integration test file?

Reply:
@bnchrch  Sort of... For instance: https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/destination-snowflake/src/test-integration/java/io/airbyte/integrations/destination/snowflake/typing_deduping/SnowflakeInternalStagingCaseInsensitiveTypingDedupingTest.java#18

Some other are referenced in docs.
If they exists there it means that in any case:
* They pass the `check` call
* They are already fetched and mounted for tests as ci_credentials would have fetched them according to the secret label

Reply:
Correct me if I'm wrong but this seems like something for the destinations team to worry about. FWIW the number of secrets seems plausible to me.

Reply:
number of secrets is a bit crazy, I'll admit. Not sure why that is.
Do we have doc around the meaning of the alias on a testSecret field? I'm not clear if we modify the way we should add new secrets to a connector either.

Reply:
@stephane-airbyte 

> number of secrets is a bit crazy, I'll admit. Not sure why that is.

These are all the secrets declared for this connector on GSM at the moment... You can remove unused one from the list safely if they're not used in your tests.

> Do we have doc around the meaning of the alias on a testSecret field? 

Not yet, it's coming. The alias is just a way to not expose the GCP project id where the secret store is. A mapping between aliases and project id will be kept in a. GHA secret.

The slight change in the way we add secret to a connector is that following the creation of a secret in GSM you'd have to update the metadata file to declare a pointer to this secret so that `airbyte-ci` will fetch them .

Reply:
I'm mostly asking because I have a PR in flight that adds a new secret to destination-s3: https://github.com/airbytehq/airbyte/pull/38204
The secrets are already in GSM, so you can add them in this PR as well